test/system/constants.js

```diff
@@ -3,8 +3,11 @@ module.exports.keepTokenAddress = "0x85Eee30c52B0b379b046Fb0F85F4f3Dc3009aFEC"
 module.exports.tbtcDepositTokenAddress =
   "0x10b66bd1e3b5a936b7f8dbc5976004311037cdf0"
 // https://allthekeeps.com/deposit/0x55d8b1dd88e60d12c81b5479186c15d07555db9d)
-module.exports.depositAddress = "0x55d8b1dd88e60d12c81b5479186c15d07555db9d"
-module.exports.bidderAddress = "0xa0216ED2202459068a750bDf74063f677613DA34"
+module.exports.depositAddress1 = "0x55d8b1dd88e60d12c81b5479186c15d07555db9d"
+module.exports.depositAddress2 = "0x8495732aecd7f132eaab61f64858ccc73475973f"
+module.exports.depositAddress3 = "0xfc9c50fd44879bd7085edd311bc8e2b7d3e41595"
```

Comment:
Would be good to note in a comment what are the differences between those deposits and when I'd choose which one.

Reply:
Sure, added description here https://github.com/keep-network/coverage-pools/pull/113